docs/pages/blog/turbo-0-4-0.mdx

```diff
@@ -88,29 +88,29 @@ RUN npm i -g turbo
 
 # Prune the workspace for the `frontend` app
 FROM base as pruner
-COPY .git ./.git
+WORKDIR /app
 COPY . .
```

Comment:
`COPY . .` will copy `.git` into `./.git`. If this was intentional for verbosity then I can drop this change.

Reply:
👍🏼 this works